GVFS/GVFS.Platform.Mac/MacPlatform.Shared.cs

```diff
@@ -17,6 +19,16 @@ public static string GetDataRootForGVFSImplementation()
         public static string GetDataRootForGVFSComponentImplementation(string componentName)
         {
             return Path.Combine(GetDataRootForGVFSImplementation(), componentName);
+        }
+
+        public static bool TryGetGVFSEnlistmentRootImplementation(string directory, out string enlistmentRoot, out string errorMessage)
+        {
+            return POSIX.POSIXPlatform.TryGetGVFSEnlistmentRootImplementation(directory, DotGVFSRoot, out enlistmentRoot, out errorMessage);
```

Comment:
Rather than `POSIX.POSIXPlatform`, could you add a `using GVFS.Platform.POSIX` and just use `POSIXPlatform` here.